docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -405,18 +463,35 @@ Check out our [advanced alerting tutorial](http://grafana.com/tutorials/alerting
 
 {{< /docs/ignore >}}
```

Comment:
Just wanted to check to see if this was a `{{< /docs/ignore >}}` without a starting tag.

I also noticed a few short codes the transformer won't support yet :
```
{{< youtube id=“ClLp-iSoaSY” >}}
{{< admonition type=“note” >}}
```
If we can wrap these in an interactive ignore that would be awsome!